### PR TITLE
make the Chinese characters indent correctly

### LINKED ISSUE #1400: Characters indent not correctly always.
**Describe the bug(__must be provided__)**
The table output not indent correctly if contains wide-size characters.
(user@127.0.0.1) [test]> fetch prop on building uuid('pin_测试') yield building.name
=================
| building.name |
=================
| 测试        |
-----------------
Got 1 rows (Time spent: 6454/7737 us)

**Your Environments(__must be provided__)**
  * OS: Linux hp-server 4.18.16-300.fc29.x86_64 #1 SMP Sat Oct 20 23:24:08 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

  * Compliler: 
g++ (GCC) 8.2.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

  * CPU: 
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              40
On-line CPU(s) list: 0-39
Thread(s) per core:  2
Core(s) per socket:  10
Socket(s):           2
NUMA node(s):        2
Vendor ID:           GenuineIntel
CPU family:          6
Model:               62
Model name:          Intel(R) Xeon(R) CPU E5-2690 v2 @ 3.00GHz
Stepping:            4
CPU MHz:             1629.598
CPU max MHz:         3600.0000
CPU min MHz:         1200.0000
BogoMIPS:            5985.55
Virtualization:      VT-x
L1d cache:           32K
L1i cache:           32K
L2 cache:            256K
L3 cache:            25600K
NUMA node0 CPU(s):   0-9,20-29
NUMA node1 CPU(s):   10-19,30-39
Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm cpuid_fault epb pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid fsgsbase smep erms xsaveopt dtherm ida arat pln pts flush_l1d

**How To Reproduce(__must be provided__)**
Fetch the result contains wide-size characters.

**Expected behavior**
Indent correctly based on characters instead of bytes.

**Additional context**
Maybe we need support the inner encoding format for characters such as utf-8, ANSI etc.

### PR DESCRIPTION
**Pull request by zhangguoqing**
close #1400

### CODE REVIEW

**Comment by Shylock-Hg** [comment]
Can't assume the encoding is utf-8 which depends on system in current version.

File: src/console/CmdProcessor.cpp [hunk 1]
```diff
@@ -109,7 +110,9 @@ void CmdProcessor::calColumnWidths(
                     break;
                 }
                 case cpp2::ColumnValue::Type::str: {
-                    size_t len = col.get_str().size();
+                    size_t wlen = utf8Conv.from_bytes(col.get_str()).size();
+                    size_t diff = (col.get_str().size() - wlen) / 2;
+                    size_t len = col.get_str().size() - diff;
```
Comment (panda-sheep) [inline src/console/CmdProcessor.cpp]: utf8 to gbk ?

Comment (panda-sheep) [reply src/console/CmdProcessor.cpp]: Later, I studied it carefully. 
The logic here should be the conversion from the byte number of the utf8 string to the byte number of the gbk string.
why utf8 to gbk ?

**Comment by dangleptr** [comment]
Close it.  @Shylock-Hg has reimplemented the console in 2.0 with GO.

**Pull request closed without merging**